### PR TITLE
Port from separate repo.

### PR DESCRIPTION
**Pull request by rgwozdz**
This is the initial port of files/code from the origin repo.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+- "8"
+sudo: false
+cache:
+- node_modules
+script:
+- npm test
+env:
+  global:
+  - NODE_ENV=test
+  - GOOGLE_CLIENT_EMAIL=dGVzdAo=
```
Comment (dmfenton) [inline .travis.yml]: are these fake?

Comment (rgwozdz) [reply .travis.yml]: Yes.  They are the string "test" base64 encoded.

Comment (dmfenton) [reply .travis.yml]: even if that's the case I think this is better off set as an ENV in travis, just so noone is tempted to replace these or ask questions in the future

Comment (rgwozdz) [reply .travis.yml]: Done in 8515e8d.

**Review by dmfenton — APPROVED** [review]

**Pull request merged**